### PR TITLE
Supply example on Document baggage propagation

### PR DESCRIPTION
**Pull request by pegasas**
try to resolve: https://github.com/open-telemetry/opentelemetry-python/issues/3561

### CODE REVIEW

**Review by cartermp — CHANGES_REQUESTED** [review]
Could you move this to the manual instrumentation page for the python docs? We don't have any language-specific content in the `/concepts` folder right now because we want to document concerns that are cross-cutting for all languages.

**Comment by pegasas** [comment]
> Could you move this to the manual instrumentation page for the python docs? We don't have any language-specific content in the `/concepts` folder right now because we want to document concerns that are cross-cutting for all languages.

you are right. I will refer javascript document for implementation.

**Comment by svrnm** [comment]
@pegasas any updates?

**Comment by pegasas** [comment]
> @pegasas any updates?

I was digging context propagation last week.
I will setup a page like https://opentelemetry.io/docs/languages/js/propagation/

**Comment by svrnm** [comment]
> > @pegasas any updates?
> 
> I was digging context propagation last week. I will setup a page like [opentelemetry.io/docs/languages/js/propagation](https://opentelemetry.io/docs/languages/js/propagation/)

thank you

File: content/en/docs/instrumentation/python/propagation.md
Comment (svrnm) [inline content/en/docs/instrumentation/python/propagation.md:1]: remove this file, we renamed `instrumentation` to `languages` so only the second is needed

File: content/en/docs/instrumentation/python/propagation.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+app = Flask(__name__)
+
+trace.set_tracer_provider(TracerProvider())
+trace.get_tracer_provider().add_span_processor(BatchSpanProcessor(ConsoleSpanExporter()))
+
+jaeger_exporter = JaegerExporter(
+    agent_host_name="localhost",  # Set your Jaeger agent host
+    agent_port=6831,  # Set your Jaeger agent port
+)
```
Comment (svrnm) [inline content/en/docs/instrumentation/python/propagation.md:41]: JaegerExporter is deprecated, please use OTLP

File: content/en/docs/languages/python/propagation.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+app = Flask(__name__)
+
+trace.set_tracer_provider(TracerProvider())
+trace.get_tracer_provider().add_span_processor(BatchSpanProcessor(ConsoleSpanExporter()))
+
+jaeger_exporter = JaegerExporter(
+    agent_host_name="localhost",  # Set your Jaeger agent host
+    agent_port=6831,  # Set your Jaeger agent port
+)
```
Comment (svrnm) [inline content/en/docs/languages/python/propagation.md:49]: JaegerExporter is deprecated, please use OTLP

File: content/en/docs/languages/python/propagation.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+from flask import Flask, request
+from opentelemetry import trace, baggage
+from opentelemetry.instrumentation.flask import FlaskInstrumentor
+ Check warning on line 207 in content/en/docs/concepts/signals/traces.md
+
+GitHub
+Actions
+/ SPELLING check
+
+Unknown word (Instrumentor)
```
Comment (svrnm) [inline content/en/docs/languages/python/propagation.md:84]: this seems to be a copy and paste issue

File: content/en/docs/languages/python/propagation.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+app = Flask(__name__)
+
+trace.set_tracer_provider(TracerProvider())
+trace.get_tracer_provider().add_span_processor(BatchSpanProcessor(ConsoleSpanExporter()))
+
+jaeger_exporter = JaegerExporter(
+    agent_host_name="localhost",  # Set your Jaeger agent host
+    agent_port=6831,  # Set your Jaeger agent port
+)
```
Comment (svrnm) [inline content/en/docs/languages/python/propagation.md:100]: same here, use OTLP

File: content/en/docs/languages/python/propagation.md (hunk 2 above)
Comment (pegasas) [reply content/en/docs/languages/python/propagation.md:49]: Done.

File: content/en/docs/languages/python/propagation.md (hunk 3 above)
Comment (pegasas) [reply content/en/docs/languages/python/propagation.md:84]: Done.

File: content/en/docs/languages/python/propagation.md (hunk 4 above)
Comment (pegasas) [reply content/en/docs/languages/python/propagation.md:100]: Done.

File: content/en/docs/instrumentation/python/propagation.md
Comment (pegasas) [reply content/en/docs/instrumentation/python/propagation.md:1]: Done.

**Comment by pegasas** [comment]
image
 
it looks good on my local environment.

**Review by cartermp — APPROVED** [review]

**Review by svrnm — APPROVED** [review]

**Comment by svrnm** [comment]
@open-telemetry/python-approvers PTAL!

File: content/en/docs/languages/python/propagation.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+trace.get_tracer_provider().add_span_processor(BatchSpanProcessor(ConsoleSpanExporter()))
+
+tracer = trace.get_tracer(__name__)
+
+@app.route('/')
+def hello():
+    # Example: Log headers received in the request in API 2
+    headers = dict(request.headers)
+    print(f"Received headers: {headers}")
+    carrier ={'traceparent': headers['Traceparent']}
```
Comment (srikanthccv) [inline content/en/docs/languages/python/propagation.md:77]: headers are case insensitive so passing the headers directly as carrier should work

**Review by srikanthccv — APPROVED** [review]

File: content/en/docs/languages/python/propagation.md (hunk 5 above)
Comment (pegasas) [reply content/en/docs/languages/python/propagation.md:77]: image
it seems like directly passing headers not working on my local testing. Should I try another Propagator?

**Comment by cartermp** [comment]
Will merge as-is, feel free to follow up if there's more to adjust

File: content/en/docs/languages/python/propagation.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
+---
+title: Propagation
+description: Context propagation for the Python SDK
+aliases: [api/propagation]
```
Comment (chalin) [inline content/en/docs/languages/python/propagation.md:4]: This is a new page, we shouldn't need an alias. Here's a followup PR to clean this up:

- #3887

**Pull request merged**